var/spack/repos/builtin/packages/py-elephant/package.py

```diff
@@ -10,20 +10,45 @@ class PyElephant(PythonPackage):
     """Elephant is a package for analysis of electrophysiology data in Python
     """
 
-    homepage = "https://neuralensemble.org/elephant"
-    pypi = "elephant/elephant-0.3.0.tar.gz"
+    homepage = "http://neuralensemble.org/elephant/"
+    pypi = "elephant/elephant-0.11.0.tar.gz"
+    git = "https://github.com/NeuralEnsemble/elephant.git"
 
+    # list of GitHub accounts to notify when the package is updated.
+    maintainers = ['Moritz-Alexander-Kern']
+
+    version('0.11.1', sha256='d604a202583440fdf9d95d42cef50a410bd74fcaaa1a925b139435f27ab012ef')
+    version('0.11.0', sha256='7b547964dbd196361edc922db2c5a7c0c886ef1effcca6c6dc7adb06f966a3be')
+    version('0.10.0', sha256='7e69a113475e4db01b3563328953c037d37f1597d9f2edf0d51fb65e9aebf096')
+    version('0.9.0', sha256='3e3d4a8e45d708f48bdcadcc4933c66f757d1ede6a1e172af0c07331b64ca180')
+    version('0.8.0', sha256='f7c2649d5b7cfdbaa4442457c75f86af01cc8e7ce2c63f5b3d4687bb94e10af2')
+    version('0.7.0', sha256='76785fe10c40042504928fde2fc57182230bbe39cf0fb0dcaffaba76219b046a')
+    version('0.6.4', sha256='b8c5f2c00ad3249e1fe428d0b8a1dbcaee4a69464481f5f8fd55d2f7f22c45a3')
     version('0.4.1', sha256='86b21a44cbacdc09a6ba6f51738dcd5b42ecd553d73acb29f71a0be7c82eac81')
     version('0.3.0', sha256='747251ccfb5820bdead6391411b5faf205b4ddf3ababaefe865f50b16540cfef')
 
     variant('doc', default=False, description='Build the documentation')
```

Comment:
```suggestion
    variant('docs', default=False, description='Install documentation dependencies')
```
This doesn't actually build the docs, it just makes sure the deps are installed. Also, let's call it `+docs` to match `pip install elephant[docs]`